README.md

```diff
@@ -1,6 +1,8 @@
 # vim-copilot-ignore
 
-A Vim plugin to automatically disable Copilot in buffers matching patterns specified in `.copilotignore` or `~/.copilotignore`.
+A Vim plugin to automatically disable Copilot in buffers matching patterns specified in a `.copilotignore` of the current work dir or a global `~/.copilotignore`.
+Here `or` is meant non-exclusively, that is, the patterns of both files are checked, and the glob patterns are documented at `:help wildcards`.
+The patterns of the global ignore file check for the whole file path whereas that of the local one for that relative to the folder of it.
```

Comment:
Not sure if I understand this last line. Perhaps an example or rewording would make it clearer.